knowledge_base/job_with_custom_sys_path/src/print.ipynb

```diff
@@ -0,0 +1,49 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The below cell retrieves the path to this bundle's deployment file path,\n",
+    "and adds it to the Python path."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "from databricks.sdk.runtime import dbutils\n",
```

Comment:
Do we need to import dbutils from the SDK here? It seems like this notebook will only be run from the workspace.